components/validator/resources.rst

```diff
@@ -71,7 +71,7 @@ configure the locations of these files::
     use Symfony\Component\Validator\Validation;
 
     $validator = Validation::createValidatorBuilder()
-        ->addYamlMapping('config/validation.yaml')
```

Comment:
I don't agree with the changes in this file: it's meant to show standalone usage of the component (a framework user will *never* write this code). So let's use `config/validation.yaml` (or even maybe `validator/validation.yaml` to be completely different)

Reply:
I updated the code parts.